CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
+
+- &#8203;<!-- 43 -->[Alert] Add `slots` and `slotProps` type to theme (#41324) @sai6855
+- &#8203;<!-- 41 -->[Autocomplete] Fix the options list being added to the DOM in `freeSolo` mode even when there are no options, causing style problems (#41300) @rakeshmusturi
+- &#8203;<!-- 15 --> Add `paperChannel` token (#41447) @siriwatknp
+- &#8203;<!-- 13 -->[Switch] Convert to support CSS extraction (#41367) @alexfauquette
+
+### `@mui/system@5.15.13`
+
+- &#8203;<!-- 03 -->[RtlProvider] Add component & hook (#41241) @mnajdova
+
+### `@mui/codemod@5.15.13`
+
+- &#8203;<!-- 12 -->[mui-codemod] Fix merging of slotProps and componentProps (#41323) @sai6855
+
+### `@mui/base@5.0.0-beta.39`
+
+- &#8203;<!-- 42 -->[material-ui][joy-ui][Autocomplete] Keep in sync highlighted index when the option still exists (#41306) @CGNonofr
+- &#8203;<!-- 40 -->[FormControl] Export `FormControlOwnerState` type from index (#41287) @michaeldfoley
+- &#8203;<!-- 39 -->[material-ui][TextareaAutosize] Fix inline style not getting applied (#41369) @ZeeshanTamboli
+
+### `@pigment-css/react@0.0.2`
+
+- &#8203;<!-- 10 -->[pigment] Handle more scenarios while transforming sx prop (#41372) @brijeshb42
+- &#8203;<!-- 09 -->[pigment] Improve testing of fixtures (#41389) @brijeshb42
+- &#8203;<!-- 07 -->[pigment-css] Fix `keyframes` serialize styles error (#41395) @siriwatknp
+- &#8203;<!-- 04 -->[pigment-css] Use class selector instead of class value (#41442) @brijeshb42
+
+### Docs
+
+- &#8203;<!-- 14 -->[material-ui] Refine checkout template (#40967) @zanivan
+- &#8203;<!-- 33 --> Finish brand name fixes #41438 @oliviertassinari
+- &#8203;<!-- 32 --> Remove noreferrer @oliviertassinari
+- &#8203;<!-- 31 --> Fix v4 docs <b> appearing in notifications (#41390) @peterwangsc
+- &#8203;<!-- 26 -->[joy-ui] Remove unnecessary styles in color inversion footer demo (#41419) @cipherlogs
+- &#8203;<!-- 25 -->[joy-ui] Update case studies chip background color (#41413) @cipherlogs
+- &#8203;<!-- 24 -->[joy-ui] Remove wrong CSS prop from the Sign-in-side template (#41383) @cipherlogs
+- &#8203;<!-- 23 -->[joy-ui] Fix broken link on the Color Inversion page (#41407) @cipherlogs
+- &#8203;<!-- 22 -->[material-ui] Adjust the Templates card design (#41450) @danilo-leal
+- &#8203;<!-- 21 -->[pigment] Add example and guide section (#41249) @siriwatknp
+- &#8203;<!-- 20 -->[pigment-css] Brand name nonbreaking space (#41438) @oliviertassinari
+- &#8203;<!-- 19 -->[pigment-css] Fix import on the README (#41411) @danilo-leal
+- &#8203;<!-- 18 -->[pigment-css] Edit starter template links on README (#41409) @danilo-leal
+- &#8203;<!-- 05 -->[pigment-css] Tweak the examples and edit READMEs (#41408) @danilo-leal
+
+### Core
+
+- &#8203;<!-- 08 -->[pigment] Make all Pigment CSS packages public (#41404) @brijeshb42
+- &#8203;<!-- 11 -->[pigment] Rename directories to match package names (#41453) @brijeshb42
+- &#8203;<!-- 06 -->[pigment-css] Example fix leading spaces (#41439) @oliviertassinari
```

Comment:
I think these ones should be under Pigment?

Reply:
I'm not sure, AFAIK none of these affect what's exposed as a package. I think they fit better here.